lib/Doctrine/DBAL/Schema/OracleSchemaManager.php

```diff
@@ -102,7 +102,7 @@ protected function _getPortableTableColumnDefinition($tableColumn)
 
         $dbType = strtolower($tableColumn['data_type']);
         if (strpos($dbType, "timestamp(") === 0) {
-            if (strpos($dbType, "WITH TIME ZONE")) {
+            if (strpos($dbType, "with time zone")) {
```

Comment:
This is is covered by the functional test case btw. Not sure how else to test this :>